docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -0,0 +1,72 @@
+
+* There are several dependencies on Amsterdam services. Replicators need this to be configurable.
+* Technical roadmap is not published.
+* [Workflow](https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/signals/workflow.py) is currently hardcoded. Names of key states can be found elsewhere, need abstraction plan.
+* Email messages are [hardcoded](https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/email_integrations/core/messages.py).
+* Separate UI text from API model classes: [example](signals/api/app/signals/apps/signals/models/history.py).
+* Extended category questions currently live in front-end code, extract and make plugable.
+* Rename or alias "sia" prefixes to something less AMS specific (tiny nitpick).
+
+
+## [Welcome contributors](https://standard.publiccode.net/criteria/open-to-contributions.html)
+
+* Improve [CONTRIBUTING.md](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md), perhaps with content from Amsterdam's [CONTRIBUTING.md](https://github.com/Amsterdam/amsterdam.github.io/blob/master/CONTRIBUTING.md)
+
+## [Maintain version control](https://standard.publiccode.net/criteria/version-control-and-history.html)
+
+* Good.
+    * Little note: not all commits state what they do or why, some just reference an issue, or have "[WIP](https://github.com/Amsterdam/signals/commit/2f9e2f73ee5dc4cdf67d6854d1a7361f6e6aaf9b)" commit messages.
+
+
+## [Require review of contributions](https://standard.publiccode.net/criteria/require-review.html)
+
+* Ensure commits are reviewed including commit message.
+
+## [Document your objectives](https://standard.publiccode.net/criteria/document-objectives.html)
+
+* Feature roadmap is not published.
+* Improve goals documented in README to include information on scope, purposes, and reasoning.
+
+## [Use plain English](https://standard.publiccode.net/criteria/understandable-english-first.html)
+
+* Translate the Dutch-only documentation and text in code, so it is also available in English.
+* Ensure dutch names, messages, words in code are localizable.
+
+## [Use open standards](https://standard.publiccode.net/criteria/open-standards.html)
+
+* Good.
+
+## [Use continuous integration](https://standard.publiccode.net/criteria/continuous-integration.html)
+
+* Tests not currently invoked by Travis or similar public continuous integration.
+* No clear integration testing or capacity testing.
+* Not clear how to do testing against a local deployment.
+* Code coverage by tests is not published.
+* Ensure test fixtures are separated from production code.
+
+## [Publish with an open license](https://standard.publiccode.net/criteria/open-licenses.html)
+
+* Good.
+
+## [Use a coherent style](https://standard.publiccode.net/criteria/style.html)
+
+* Conforms to the PEP8 style guide, which can be automatically checked; add to Continuous integration.
```

Comment:
This is already checked when running the tests. So that would be solved when the test suite will run outside of the Datapunt CI, right?

Reply:
Yes, certainly.